java/bogdana_maksiuta/src/com/zoolatech/lecture10/tasks/_1/Main.java

```diff
@@ -0,0 +1,64 @@
+package com.zoolatech.lecture10.tasks._1;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Scanner;
+
+/**
+ * Given a directory called db with 3 files:
+ * names (Ex. of content Alex, John, Maria)
+ * surnames (Scott, Fung, Chan)
+ * positions (Delivery Manager, Senior Java Dev, Product Manager)
+ * <p>
+ * When user starts app it can see in terminal list of files without content.
+ * User can type file name like "names" and see content of the file on terminal.
+ * If user type invalid filename app should write message to err stream,
+ * draw file names again and write invalid filename to a new file called "error.log" (ideally data should be appended).
+ */
+
+public class Main {
+    static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        String directionToTaskFolder = "src/com/zoolatech/lecture10/tasks/_1/";
+        File folder = new File(directionToTaskFolder + "db");
+
+        ArrayList<String> filesList = new ArrayList<>();
+        for (File file : Objects.requireNonNull(folder.listFiles())) {
```

Comment:
nit: I like that you added a check here =) I'd check not in a loop. One line before that.
But please pay attention to that it's not a good design in java. To avoid NulllPointerExceptions never return null if the return time is collection. Always return empty collection to avoid such checks on client side